packages/nimble-components/src/rich-text/mention-listbox/tests/mention-listbox.spec.ts

```diff
@@ -45,15 +49,24 @@ describe('RichTextMentionListbox', () => {
         await waitAnimationFrame(); // necessary because scrolling is queued with requestAnimationFrame
     }
 
+    async function showAndWaitForOpen(
+        listbox: RichTextMentionListbox,
+        anchor: HTMLElement
+    ): Promise<void> {
+        const regionLoadedListener = createEventListener(listbox, 'loaded');
+        listbox.show({
+            anchorNode: anchor,
+            filter: ''
+        });
+        await regionLoadedListener.promise;
+    }
+
     // Intermittent: see https://github.com/ni/nimble/issues/1891
     xit('should scroll the selected option into view when opened', async () => {
         const model = new Model();
         const { connect, disconnect } = await setup500Options(model);
         await connect();
-        model.mentionListbox.show({
-            anchorNode: model.anchorDiv,
-            filter: ''
-        });
+        await showAndWaitForOpen(model.mentionListbox, model.anchorDiv);
```

Comment:
I'm not sure if waiting for the anchored region's `loaded` event will fix the intermittency here, but I updated this test for consistency.

Reply:
Would it be worth re-enabling the test and seeing if the intermittency is gone now?

Reply:
Guess it couldn't hurt. Let's do it.